index.bs

```diff
@@ -8041,63 +8043,367 @@ ECMAScript Object values.
 
 <h4 id="es-promise">Promise types — Promise&lt;|T|&gt;</h4>
 
-IDL [=promise type=] values are
-represented by ECMAScript {{ECMAScript/Promise}}
-objects.
+IDL [=promise type=] values are represented by ECMAScript [=PromiseCapability=] records.
 
 <div id="es-to-promise" algorithm="convert an ECMAScript value to promise">
 
     An ECMAScript value |V| is [=converted to an IDL value|converted=]
-    to an IDL <a interface lt="Promise">Promise<var ignore>T</var></a> value as follows:
+    to an IDL <code>{{Promise}}&lt;<var ignore>T</var>></code> value as follows:
```

Comment:
This pattern results in nested `<code>`. I guess that's not technically invalid? But what would work better would be the verbose

```html
<code><a interface>Promise</a>&lt;<var ignore>T</var>></code>
```